app/models/package.rb

```diff
@@ -60,6 +60,7 @@ class Package < ApplicationRecord
   end
 
   validates :package_type_id, presence: true
+  validates :notes, presence: true
```

Comment:
Migration note: there are 413 packages where `notes = NULL` and 84 packages `where notes = ''` so we will need to fill these in before migrating. Easiest option is to copy across the code description. Can be done in SQL.

Can you add a postgresql constraint as well?

Reply:
Sure. I have added it